File: src/main/java/pl/yalgrin/playnite/simplesync/mapper/objects/AgeRatingMapper.java
package pl.yalgrin.playnite.simplesync.mapper.objects;

import org.springframework.stereotype.Component;
import pl.yalgrin.playnite.simplesync.library.domain.AgeRating;
import pl.yalgrin.playnite.simplesync.library.dto.AgeRatingDTO;

@Component
public class AgeRatingMapper extends AbstractObjectMapper<AgeRating, AgeRatingDTO> {

    @Override
    protected AgeRatingDTO createDTO() {
        return new AgeRatingDTO();
    }
}

File: src/main/kotlin/pl/yalgrin/playnite/simplesync/library/mapper/AgeRatingMapper.kt
package pl.yalgrin.playnite.simplesync.library.mapper

import org.springframework.stereotype.Component
import pl.yalgrin.playnite.simplesync.library.domain.AgeRating
import pl.yalgrin.playnite.simplesync.library.dto.AgeRatingDTO

@Component
class AgeRatingMapper : LibraryObjectMapperImpl<AgeRating, AgeRatingDTO>() {
    override fun createDTO(): AgeRatingDTO = AgeRatingDTO()
}